bot/exts/evergreen/duck_game.py

```diff
@@ -0,0 +1,329 @@
+import asyncio
+import random
+import re
+from collections import defaultdict
+from io import BytesIO
+from itertools import product
+from pathlib import Path
+from urllib.parse import urlparse
+
+import discord
+from PIL import Image, ImageDraw, ImageFont
+from discord.ext import commands
+
+from bot.bot import Bot
+from bot.constants import Colours, MODERATION_ROLES
+
+
+DECK = list(product(*[(0, 1, 2)]*4))
+
+GAME_DURATION = 180
+
+# Scoring
+CORRECT_SOLN = 1
+INCORRECT_SOLN = -1
+CORRECT_GOOSE = 2
+INCORRECT_GOOSE = -1
+
+""" Distribution of minimum acceptable solutions at board generation.
+    This is for gameplay reasons, to shift the number of solutions per board up,
+    while still making the end of the game unpredictable.
+    Note: this is *not* the same as the distribution of number of solutions.
+"""
```

Comment:
Block comments should be lines starting with `#`s rather than a docstring, as per pep 8 https://www.python.org/dev/peps/pep-0008/#block-comments